README.md

```diff
@@ -81,43 +149,25 @@ The library provides additional utilities to ensure the best results for the tas
   let { red, bgWhite } = pc.createColors(options.enableColors);
   ```
 
-## Benchmarks
+## Replacing `chalk`
 
-`nanocolors` [benchmark](https://github.com/ai/nanocolors/tree/main/test):
+1. Replace import and use named exports:
```

Comment:
Not named anymore?

Reply:
We improved benchmarks a lot. `picocolors`’s benchmarks are now much closer to the reality.

We found that text size affects benchmarks and tests like `red('test')` doesn’t show the real performance. Now `complex` benchmark is try to combine different use cases to be closer to the reality.

Reply:
`nanocolors` will be mentioned in Prior Art. We do not need to put it to benchmark results.

Reply:
No, I mean picocolors doesn't have named exports anymore, right?
> Replace import and use named exports:

Reply:
Fixed 😅